src/plugins/irc-events/link.js

```diff
@@ -1,35 +1,26 @@
 "use strict";
 
 const cheerio = require("cheerio");
-const Msg = require("../../models/msg");
 const request = require("request");
 const Helper = require("../../helper");
+const findLinks = require("../../../client/js/libs/handlebars/ircmessageparser/findLinks");
```

Comment:
Actually, I just realized (:man_facepalming:) that this server file is loading a client file. Any way we can make this a bit cleaner? Not sure what's the best way, I'm just wondering.